content/community/changelog/app-frontend/v3/whats-new/_index.nb.md

```diff
@@ -3,6 +3,12 @@ title: Hva er nytt?
 description: Oversikt over endringer som ble introdusert i v3 av app frontend.
 toc: true
 ---
+
+## 3.23.0 (2022-01-20) - Lagt til flere datakilder for variabler i tekst
+Funksjonaliteten knyttet til variabler i tekster har fått to nye datakilder. Det blir nå mulig å hente verdier fra instance og konfigurasjonsverdier fra ApplicationSettings. Støtten for konfigurasjonsverdier krever versjon 4.23.0 eller nyere av backend sine NuGet pakker.
```

Comment:
Blir trolig 4.25.0 slik ting ser ut nå 

```suggestion
Funksjonaliteten knyttet til variabler i tekster har fått to nye datakilder. Det blir nå mulig å hente verdier fra instance og konfigurasjonsverdier fra ApplicationSettings. Støtten for konfigurasjonsverdier krever versjon 4.25.0 eller nyere av backend sine NuGet pakker.
```

Reply:
I suspected that this would happen. Fixing.